megamek/src/megamek/common/Aero.java

```diff
@@ -3375,6 +3373,14 @@ public void setVSTOL(boolean b) {
         vstol = b;
     }
 
+    public double getVSTOLGearTonnage(){
```

Comment:
This duplicates code already present in TestAero. Where possible, calculations related to construction rules, when not required to load the unit or required during gameplay, should be placed the megamek.common.verifier package rather than the Entity. This will also report an incorrect value for FixedWingSupport, which adds VSTOL capability through a chassis modification.

Reply:
Good to know, and fixed.